### PR TITLE
git actions upgrade + add trivy

### PR DESCRIPTION
**Pull request by laxmanchekka**
## Description
git actions upgrade + add trivy

### CODE REVIEW

File: .github/workflows/pr-build.yml [hunk 1]
```diff
@@ -24,17 +24,19 @@ jobs:
...
+        id: trivy-scan-kafka
         with:
           image: hypertrace/kafka
           output-mode: github
+          category: kafka
 
       - name: Run Trivy vulnerability scanner for kafka-zookeeper image
         uses: hypertrace/github-actions/trivy-image-scan@main
-        id: trivy-scan-2
+        id: trivy-scan-kafka-zk
```
Comment (ravisingal) [inline .github/workflows/pr-build.yml:35]: `id` is not required.

File: .trivyignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# org.scala-lang:scala-library (from upstream opensource dependency)
```
Comment (aaron-steinfeld) [inline .trivyignore:1]: Can we constraint it? Guessing yes given the age.

File: .trivyignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+CVE-2022-36944 exp:2023-09-30
+
+# org.xerial.snappy:snappy-java (from upstream opensource dependency)
```
Comment (aaron-steinfeld) [inline .trivyignore:4]: This one we can definitely fix, perhaps use the kafka bom or just constraint here directly

Comment (laxmanchekka) [reply .trivyignore:4]: This is kafka server image. Not kafka-streams application. We don't have any codebase and don't build the code. We just use apache tarball distro to build our image.

File: .trivyignore (hunk 2 above)
Comment (laxmanchekka) [reply .trivyignore:1]: This is kafka server image. Not kafka-streams application. We don't have any codebase and don't build the code. We just use apache tarball distro to build our image.

Comment (aaron-steinfeld) [reply .trivyignore:1]: got it.

File: Dockerfile [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # TODO: port this similarly to kafka-zookeeper in order to share base layer
 # The only assumption we make about this FROM is that it has a JRE in path
-FROM adoptopenjdk/openjdk11:jre-11.0.18_10@sha256:3282670f5b315c731cb69d62b5d7eb195392be4a55d22a398c9ed1008490de7d
+FROM adoptopenjdk/openjdk11:jre-11.0.19_7@sha256:34154cd2f45cddfccd14847e2fdbaba72261e3ee5fa205b7426efacb0904e46b
```
Comment (laxmanchekka) [inline Dockerfile:3]:
```suggestion
FROM adoptopenjdk/openjdk11:jre-11.0.19_7@sha256: a2160bc43c40d98324f43260344bf70f3a1d6a408d5dc005d2e6fb4f0f788ec4
```

**Review by ravisingal — APPROVED** [review]

**Pull request merged**